docs/examples/alibi-detect/README.md

```diff
@@ -75,35 +106,35 @@ For that, we will need to create 2 configuration files:
 }
 ```
 
+    Overwriting settings.json
```

Comment:
Should we clear the outputs from the notebook? Again, not a big deal (i.e. feel free to skip this one).

Reply:
Fixed now in 8cdec90